android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AuthorizationsDaoInterface.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Solana Mobile Inc.
+ */
+
+package com.solana.mobilewalletadapter.walletlib.authorization;
+
+import androidx.annotation.NonNull;
+
+import java.util.List;
+
+/*package*/ interface AuthorizationsDaoInterface {
```

Comment:
Can we fully annotate method parameters and return types with `@Nullable` or `@NonNull`, as appropriate? Bonus points for `@Size(min=...)` and `@IntRange` where it makes sense.

Reply:
Done.

Reply:
The return types still need annotation (here and in the implementing class)